### ISSUE TITLE
Hit testing on default time app is returning wrong node.

### ISSUE THREAD

**Issue by kf6gpe** [flutter/flutter#59269]
https://bugs.fuchsia.dev/p/fuchsia/issues/detail?id=53053

/cc @dnfield

**Comment by dnfield** [comment]
Customer is indicating bug still exists. Moving milestone out while we continue to investigate root cause.

**Comment by dnfield** [comment]
Requested a minimal reproduction from customer.

**Comment by chinmaygarde** [comment]
Triage Update: There has been no progress on the reproducible test case.

**Comment by kf6gpe** [comment]
I've pushed back on this issue to get a reproducible test case, as @dnfield and @iskakaushik indicated that they cannot reproduce this with the steps given due to target hardware/build instability. Leaving as a P1, as a11y is an important issue for the team right now, but there's nothing for the Flutter team to do with this at the moment.

**Comment by chinmaygarde** [comment]
Triage: Nothing to do as there is no reproducible test case.

**Comment by kf6gpe** [comment]
@iskakaushik, when you get a moment, could you try the use case detailed as comment 21 on https://bugs.fuchsia.dev/p/fuchsia/issues/detail?id=53053 if you have running hardware and comment on the upstream issue? Thanks!

**Comment by chinmaygarde** [comment]
Triage: Still waiting for a reproducible test case. Reducing priority to match upstream prioritization.

**Comment by dnfield** [comment]
Customer is reporting this is fixed upstream.

**Comment by dnfield** [comment]
Suspicion is that the patch I landed that I thought fixed this did fix it, but that it had not rolled in yet last time there was testing.
